setup.py

```diff
@@ -40,13 +40,20 @@ def fill_connectors(projects, modules_path):
         if not os.path.isfile(os.path.join(modules_path, module, SKIP_ME)):
             projects['stix_shifter_modules_' + module] = ['stix_shifter_modules/' + module]
 
+def fill_connectors_mode3(projects, modules_path):
+    modules = [name for name in os.listdir(modules_path)
+               if (os.path.isdir(os.path.join(modules_path, name)) and (not name.startswith('__')))]
+    for module in modules:
+        if not os.path.isfile(os.path.join(modules_path, module, SKIP_ME)):
+            projects['stix_shifter_modules'].append('stix_shifter_modules/' + module)
+
 # The mode determines how the stix-shifter is packaged
 # 1 = Include everything in 1 whl package
 # 3 - 3 whl packages respectively for stix-shifter, stix-shifter-utils and stix-shifter-modules
 # N - stix-shifter, stix-shifter-utils, and each connector is packaged separately
 # <module name> - package only the specified connector
 
-mode = 'N'
+mode = '3'
```

Comment:
you should not change the default packaging mode, why are you using mode 3 at all? maybe mode 1 will fit you better?

Reply:
Agreed. Changing the default mode might be too much depending on the workflow. For the distribution purpose for developers (`python setup.py bdist_wheel`), it would be good to have separate packages.

But, for local build/install for users (`python setup.py install`), mode 3 or 1 is needed as the installed packages under `site-packages` will have the mix of `egg` zipfiles and directories for each stix_shifter_module, which will cause importing errors. For example, after `python setup.py install` (with mode `N`), importing qradar module (`import stix_shifter_modules.qradar`) from _outside_ of the stix-shifter root directory will cause `ModuleNotFoundError` due to above issues.

Reply:
By the way, MODE 1/3 bdist/install fails due to incorrect path resolution for stix_shifter_modules. Proper `fill_connectors` is needed to support MODE 1/3.

Reply:
@mdazam1942  is looking into it.